### ISSUE TITLE
[Designer] Text drawn outside of card area when using a large pixel width for columns

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#3009]
# Platform

What platform is your issue or question related to? (Delete other platforms).

* JavaScript

# Issue

When using the designer, if you set a column width to larger than the available space the text is drawn outside the card area. 

![image](https://user-images.githubusercontent.com/9091833/58737456-7911b180-83b6-11e9-837d-ece999366ba8.png)

```json
{
    "type": "AdaptiveCard",
    "body": [
        {
            "type": "ColumnSet",
            "columns": [
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
                            "wrap": true
                        }
                    ],
                    "width": "50px"
                },
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
                            "wrap": true
                        }
                    ],
                    "width": "100px"
                },
                {
                    "type": "Column",
                    "items": [
                        {
                            "type": "TextBlock",
                            "text": "Video provides a powerful way to help you prove your point. When you click Online Video, you can paste in the embed code for the video you want to add. You can also type a keyword to search online for the video that best fits your document.",
                            "wrap": true
                        }
                    ],
                    "width": "2000px"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "version": "1.0"
}
```

**Comment by dclaux** [comment]
Fix pending here: https://github.com/microsoft/AdaptiveCards/pull/3008

**Comment by shalinijoshi19** [comment]
Fixed by #3008
